classes/submission/reviewAssignment/ReviewAssignment.inc.php

```diff
@@ -21,19 +21,24 @@
 define('SUBMISSION_REVIEWER_RECOMMENDATION_DECLINE', 5);
 define('SUBMISSION_REVIEWER_RECOMMENDATION_SEE_COMMENTS', 6);
 
-define('SUBMISSION_REVIEWER_RATING_VERY_GOOD', 5);
```

Comment:
We might want to keep those constants around. See https://github.com/pkp/pkp-lib/issues/372.